src/renderer.js

```diff
@@ -86,13 +94,9 @@ Renderer.prototype.render = function () {
   gl.viewport(0, 0, gl.drawingBufferWidth, gl.drawingBufferHeight);
   gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT);
 
-  var uniforms = updateUniforms(gl, this.camera, this.uniforms);
-
-  //renderPass(gl, depthProgramInfo, faceBufferInfo, uniforms, 'TRIANGLES');
+  var uniforms = updateUniforms(gl, this.camera, this.glTexture, this.uniforms);
   renderPass(gl, this.shaders.face, this.buffers.face, uniforms, 'TRIANGLES');
-  //renderPass(gl, this.shaders.fold, this.foldBufferInfo, uniforms, 'LINES');
 
-  //renderPoints(points, pointDivs);
   this.raf = requestAnimationFrame(this.render);
```

Comment:
Resize is called on render, so I'm not sure that will work :P Doing it this way actually makes rendering from resizing much faster than listening to window resize events which can fire more often than animation frames.